pilot/pkg/model/validation.go

```diff
@@ -1460,6 +1471,16 @@ func ValidateServiceRole(name, namespace string, msg proto.Message) error {
 	return errs
 }
 
+// arePortsInRange checks if all ports in range [0, 65535]
+func arePortsInRange(ports []int32) bool {
```

Comment:
You can just reuse `ValidatePort()` in this file.

Reply:
Ah, I didn't see this function. Will do. Thanks!